libunwind/src/Registers.hpp

```diff
@@ -29,7 +29,9 @@ struct __attribute__((aligned(16))) fake_capability {
   char bytes[16];
 };
 typedef struct fake_capability fake_capability_t;
+#define _LIBUNWIND_UNDEF_CAPABILITY ((fake_capability_t){0})
```

Comment:
Should this not be {{0}}?

Reply:
I'm not sure if it makes a difference in this case, but yes it was supposed to be `{{0}}`, thanks!